### PR TITLE
skip files with whitespace on AIX/bff packaging

### PR DESCRIPTION
**Pull request by lamont-granquist**
This is the AIX version of #609

### CODE REVIEW

**Comment by lamont-granquist** [comment]
@chef/omnibus-maintainers this is necessary to get ruby-2.2.4 building+installing on AIX.

i tried adding spaces to the regex that does the file-renaming dance, but couldn't make it work.  this works, and if it ever breaks builds then solaris will be even worse anyway...

File: lib/omnibus/packagers/bff.rb [hunk 1]
```diff
@@ -131,8 +131,14 @@ def write_scripts
     def write_gen_template
 
       # Get a list of all files
-      files = FileSyncer.glob("#{staging_dir}/**/*").map do |path|
-
+      files = FileSyncer.glob("#{staging_dir}/**/*").reject do |path|
+        # remove any files with spaces.
+        if path =~ /\s+/
```
Comment (sethvargo) [inline lib/omnibus/packagers/bff.rb]: `[[:space:]]` will cover more things here

Comment (lamont-granquist) [inline lib/omnibus/packagers/bff.rb]: heh, i can't wait to see gems with checked in spec files that have unicode emojis in filenames and what those do to bff and solaris pkg packaging...

**Comment by schisamo** [comment]
I'm :+1: if @scotthain is happy.

**Comment by scotthain** [comment]
Ugh. Yep. That's a thing :+1:

**Pull request merged**